# Program 4: A python program to demostrate runtime errors
love=['Himneesh','Spoorthi','Sunitha','love','sex']
print(love[5])
'''Exception has occurred: IndexError
list index out of range
  File "C:\Users\HP\OneDrive\Desktop\python text book examples\Exceptions\Program_4.py", line 3, in <module>
    print(love[5])
          ~~~~^^^
IndexError: list index out of range'''